<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addWindow</class>
 <widget class="QDialog" name="addWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Add New Pokemon</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="ftypeLabel">
     <property name="text">
      <string>First Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameInput"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="hpLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Max HP: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="stypeLabel">
     <property name="text">
      <string>Second Type (optionally):</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="cpLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Max CP:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="cpInput">
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="hpInput">
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="ftypeInput">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>Choose first type</string>
     </property>
     <item>
      <property name="text">
       <string>Bug</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dragon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Electric</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fairy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fighting</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flying</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ghost</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Grass</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ground</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ice</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Poison</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Psychic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rock</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Water</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>Choose second type</string>
     </property>
     <item>
      <property name="text">
       <string>Bug</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dragon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Electric</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fairy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fighting</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flying</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ghost</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Grass</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ground</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ice</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Poison</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Psychic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rock</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Water</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameInput</tabstop>
  <tabstop>ftypeInput</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>cpInput</tabstop>
  <tabstop>hpInput</tabstop>
  <tabstop>addButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
